### DISCUSSION TITLE
Overlap between "For you" and "Following"

### DISCUSSION THREAD

**Discussion by manosbatsis** [community/community#36439] [Other Feature Feedback, Questions, & Ideas]
The content overlap between the two tabs is way too high. "For you" should exclude content already provided by "Following".

**Comment by ossanna16** [comment]
Hey there @manosbatsis and thank you for your feedback! Can you please add some more detail including screenshots? I'm glad you found our community 🙂

**Comment by OlgaGrom** [comment]
Hi
